tests/plotting/test_plotting.py

```diff
@@ -2334,3 +2334,16 @@ def test_ruler(sphere):
     plotter.add_ruler([-0.6, -0.6, 0], [0.6, -0.6, 0], font_size_factor=1.2)
     plotter.view_xy()
     plotter.show(before_close_callback=verify_cache_image)
+
+
+def test_plot_complex_value(plane):
+    """Test plotting complex data."""
+    data = np.arange(plane.n_points, dtype=np.complex128)
+    data += np.linspace(0, 1, plane.n_points) * -1j
+    with pytest.warns(np.ComplexWarning):
+        plane.plot(scalars=data)
+
+    pl = pyvista.Plotter()
+    with pytest.warns(np.ComplexWarning):
+        pl.add_mesh(plane, scalars=data, show_scalar_bar=False)
+    pl.show(before_close_callback=verify_cache_image)
```

Comment:
Could you please show the scalar bar with a title to make it clear what the plot is showing?

Reply:
Added